### PR TITLE
Expired flags

### PR DESCRIPTION
**Pull request by aintgoin2goa**

### CODE REVIEW

File: test/nightwatch.check.spec.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ describe('Nightwatch Check', function(){
 			nightWatchCheck = new NightWatchCheck(config);
 		});
 
-		it.skip('Should be able to get a list of jobs from saucelabs', function(done){
+		it('Should be able to get a list of jobs from saucelabs', function(done){
```
Comment (matthew-andrews) [inline test/nightwatch.check.spec.js]: :innocent:

File: config/flags.js [hunk 2]
```diff
@@ -67,6 +67,24 @@ module.exports = {
...
+			"businessImpact" : "Functionality behind expired flags will not appear on the site",
+			"technicalSummary": "Tests that there are no expired flags",
+			"panicGuide": "Don't Panic",
+			"checkResult" : {
+				"PASSED" : "There are no expired flags",
+				"FAILED" : "There are expired flags in the toggler, see them here: https://next.ft.com/__toggler/expired.html",
+				"PENDING" : "This test has not yet run"
+			},
+			interval: '60s',
+			callback : function(json){
```
Comment (matthew-andrews) [inline config/flags.js]: does this need to be implemented?

Comment (aintgoin2goa) [inline config/flags.js]: sorry what do you mean?  do we need this at all?  or are you being more specific?

Comment (matthew-andrews) [inline config/flags.js]: it's an empty function?

**Comment by matthew-andrews** [comment]
Do you think we need two health checks?

a level 2 if flags expire?  (affects customer behaviour)
and a level 3 if flags are on the list of expiring soon?

**Comment by aintgoin2goa** [comment]
Yeah why not - I wish we had something that would email the owner of the flag as well...

**Comment by aintgoin2goa** [comment]
Actually I'm not convinced.  Then we'd be failing health checks even though the system is healthy.  I think we need some other kind of monitoring app

**Comment by matthew-andrews** [comment]
yep fair enough :+1:

**Pull request merged**